ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/qos-profiles.j2

```diff
@@ -20,9 +20,9 @@ QOS Profile: **{{ profile }}**
 | {{ cos }} | {{ dscp }} | {{ trust }} | {{ shape_rate }} | {{ qos_sp }} |
 {%         if qos_profiles[profile].tx_queues is arista.avd.defined %}
 
-**Tx-queues**
+**tx-queues**
 
-| Tx-queue | Bandwidth | Priority | Shape Rate |
+| tx-queue | Bandwidth | Priority | Shape Rate |
```

Comment:
TX should be uppercase since it is an abbreviation. Adding a `type` column with `Unicast`, `Multicast` or `All` would make the documentation much simpler, and give a complete overview.